app/src/test/java/com/kickstarter/libs/utils/ExperimentUtilsTest.kt

```diff
@@ -15,7 +15,7 @@ class ExperimentUtilsTest : KSRobolectricTestCase() {
                 .backedProjectsCount(10)
                 .build()
         val experimentData = ExperimentData(user, RefTag.discovery(), RefTag.search())
-        val attributes = ExperimentUtils.attributes(experimentData, "9.9.9", "10", OptimizelyEnvironment.STAGING)
+        val attributes = ExperimentUtils.attributes(experimentData, "9.9.9", "10", OptimizelyEnvironment.DEVELOPMENT)
```

Comment:
Why this change?

Reply:
The test is checking attributes for a user not logged in production, I changed it to Development instead of staging because for staging using internally a FirebaseID, which requires have Firebase active in a unit tes and an API key, no unit test should need that.
This rise a question, why for staging we need to add the FirebaseID into this experiment params, (for production is null)? which I cannot answer tbh  😞.
You can see it here: ExperimentUtils.kt:13